addons/html_builder/static/src/core/building_blocks/select_many2x.scss

```diff
@@ -0,0 +1,70 @@
+.o-hb-selectMany2X-wrapper {
+    // Disclaimer: the following rules essentially overrides utility classes
+    // "inherited" from the backend and apply BuilderSelect rules on top.
+    // To be improved by removing the unwanted classes and share relevant
+    // rules across the two components.
+
+    border: 0 !important; // Override backend's `border`class
+    width: 100% !important; // Override backend's `w-auto`class
+
+    div:has(> &) {
+        // Apply `o-hb-select-wrapper` classes on the unnamed parent div
```

Comment:
Is this a "TODO" ? Would be indeed better to apply a class than have this kind of selector IMO :grimacing: I am not sure the "disclaimer" above says that this would be to change :upside_down_face:

Reply:
Wasn't clear indeed. Added I'll add proper "todo" comments